core/src/main/scala/ch/epfl/scala/index/newModel/NewProject.scala

```diff
@@ -11,42 +11,43 @@ case class NewProject(
     githubInfo: Option[GithubInfo],
     esId: Option[String],
     // form data
-    formData: FormData
+    dataForm: DataForm
 ) {
   val reference: Project.Reference =
     Project.Reference(organization.value, repository.value)
-  val hasCli: Boolean = formData.cliArtifacts.nonEmpty
+  val hasCli: Boolean = dataForm.cliArtifacts.nonEmpty
   //val created: datetime = firstRelease.released
   //val lastUpdated: datetime = lastReleaseAdd.released
   def update(newGithubInfo: Option[GithubInfo]): NewProject =
     copy(githubInfo = newGithubInfo)
+  def contributorsWanted: Boolean = dataForm.contributorsWanted
 }
 
 object NewProject {
   def defaultProject(
       org: String,
       repo: String,
       githubInfo: Option[GithubInfo],
-      formData: FormData = FormData.default
+      formData: DataForm = DataForm.default
   ): NewProject =
     NewProject(
       Organization(org),
       repository = Repository(repo),
       githubInfo = githubInfo,
       esId = None,
-      formData = formData
+      dataForm = formData
     )
 
-  case class FormData(
+  case class DataForm(
```

Comment:
Not sure if it's a better name. Maybe we should call them `ProjectSettings` because that's the thing the project owner can modify on its project.

Reply:
in another commit. It needs a bigger change to include database rename